### ISSUE TITLE
Cinnamenu: Allow two columns in grid mode

### ISSUE THREAD

**Issue by DarkWiller** [linuxmint/cinnamon-spices-applets#1805]
```
 * Applet version/Build date (3.2.1)
 * Cinnamon version (Cinnamon 3.6.7)
 * Distribution - (Mint 18.3)
 * Graphics hardware *and* driver used (Nvidia 920mx, Intel Core i5, 7ª Generation, 8gb Ram
 * Snippet or attachment of ~/.cinnamon/glass.log
 * Snippet or attachment of ~/.xsession-errors

 ```

 @jaszhix 

**Notify author of applet**
@jaszhix 

**Issue**  
Request for resource improvement

**Steps to reproduce** 
The application offers the option of three (3) to seven (7) columns in the icon view (columns).

**Expected behaviour**
That the application offered the option of two (2) to seven (7) columns in the icon view (columns).

**Other information**
Congratulations on the work with Cinnamenu. Excellent job.

**Comment by DarkWiller** [comment]
this is the first time I comment on github. Sorry if I did something wrong.

**Comment by DarkWiller** [comment]
Thank you for considering my appeal. Looking forward to seeing your work on this new look.

**Comment by jaszhix** [comment]
Added in #1932

**Comment by DarkWiller** [comment]
I am very happy to have considered my request. It stayed exactly as I'd like it to be. This is undoubtedly one of the best menu for linux.
Your work is worthy of the best marks.
Thank you

Willer


________________________________
De: Jason Hicks <notifications@github.com>
Enviado: quarta-feira, 22 de agosto de 2018 08:51:31
Para: linuxmint/cinnamon-spices-applets
Cc: DarkWiller; State change
Assunto: Re: [linuxmint/cinnamon-spices-applets] Cinnamenu: Allow two columns in grid mode (#1805)


Closed #1805<https://github.com/linuxmint/cinnamon-spices-applets/issues/1805>.
